services/streammanager/kafka/kafkamanager.go

```diff
@@ -312,7 +313,7 @@ func NewProducer(destination *backendconfig.DestinationT, o common.Opts) (*Produ
 		return nil, fmt.Errorf("could not ping: %w", err)
 	}
 
-	p, err := c.NewProducer(destConfig.Topic, client.ProducerConfig{
+	p, err := c.NewProducer("", client.ProducerConfig{
```

Comment:
If this is how you want to handle topics then why keeping the argument in the constructor? It looks like you'll be passing it each time anyway even for old configurations.

Reply:
Yes we are planning the same approach updated for the same